# Bezieht sich auf eine Zone, refenziert Scanner und frame

std_msgs/Header header

geometry_msgs/Polygon warn
geometry_msgs/Polygon safety

float32 speed_lower
float32 speed_upper